### ISSUE TITLE
Images are cut at the by the post's blank.

### ISSUE THREAD

**Issue by Mek101** [bleakgrey/tootle#255]
#### What should happen:
The images should be fully displayed

#### What happened instead:
The post's blank expands on the images, cutting the lower and upper sides.

#### Screenshots (if applicable):
What it is
![What it is](https://i.loli.net/2020/10/22/NPh42vUkLGH3Jy6.png)
The actual post view via web
![The actual view via web](https://i.loli.net/2020/10/22/WrX2ELvVlPOuNS3.png)

#### Build context:
 - **Distro:** Linux Mint 20.0
 - **App version:** 1.0.0-alpha
 - **Build type:** Flatpak

**Comment by Kekun** [comment]
That's an issue that annoys me quite a lot too. If images are displayed fully though, it should be up to a given ratio, e.g. we don't want images like http://longc.at/longcat to take the app over.

But I think displaying the images fully isn't exactly what we want, what we probably want is an images section that is taller than it currently is and has a constant ratio, so the way images are presented remains consistent when I resize the window, maybe 3:2 or 16:9. The images probably should be zoomed in as they are now though, to avoid empty borders. Basically, I think what's in the web view is good.

What is even more important though IMO, is to be able to fully see an image, zoom in it, etc., without opening the whole post in a web browser, it should be doable in Tootle itself, e.g. the image could be opened in a subpage thanks to a `HdyDeck` wrapping the whole window.

**Comment by Mek101** [comment]
A page for the image to be fully displayed seems the most sensible solution to me. That or displaying the image to it's full size in the post page

**Comment by bleakgrey** [comment]
> What is even more important though IMO, is to be able to fully see an image, zoom in it, etc., without opening the whole post in a web browser, it should be doable in Tootle itself

Just to be clear, the first alpha *is supposed* to download and open attachments in the default app associated with the clicked attachment type.

The URI opening function seems to _silently_ fail if no photo/video app is found, hence you may get an impression that Tootle has no way of displaying the full-scale image _whatsoever_. This can be reproduced in Flatpak and should probably be filed as a separate issue.

**Comment by XanderLeaDaren** [comment]
I agree with the original post that images should be displayed with a bigger height inside the app, to reflect the ratio used on Mastodon instances.
